services/ui-src/src/forms/mcpar/ebssei/ebssei.schema.ts

```diff
@@ -1,3 +1,9 @@
 import { object } from "yup";
+import { checkbox, nested, text } from "utils/forms/schemas";
 
-export default object({});
+export default object({
+  "ebessei-1": checkbox(),
+  "ebessei-1-13-text": nested(text, "ebessei-1-o13", "Other"),
```

Comment:
Remember to check that the validation works. The text fields are not being treated as required when present because:

  1. The field id is incorrect, and 
  2. The parent field id is incorrect.

`nested` takes the following props: `fieldSchema: Function`, `parentFieldName: string`, `parentOptionValue: any`.

Same comment for line 8 below.

```suggestion
  "ebessei-1-o13-text": nested(text, "ebessei-1", "Other"),
```